### PR TITLE
JOB-30707 Close modal when request fails

### PR DESCRIPTION
**Pull request by cwtedwards**
- set `listViewedDisplayed` to `false` when the autocomplete request doesn't have a status of 200
- set `listViewedDisplayed` to `true` when the autocomplete request is a 200 again

Implementation and more detail with this PR https://github.com/GetJobber/jobber-mobile/pull/1065

### CODE REVIEW

File: GooglePlacesAutocomplete.js [hunk 1]
```diff
@@ -525,6 +526,7 @@ export const GooglePlacesAutocomplete = forwardRef((props, ref) => {
             }
           }
         } else {
+            setListViewDisplayed(false);
```
Comment (C-Flatla) [inline GooglePlacesAutocomplete.js:529]: 💅  this is indented once too many

```suggestion
          setListViewDisplayed(false);
```

Comment (cwtedwards) [reply GooglePlacesAutocomplete.js:529]: 😭  my linter doesn't seem to want to play ball in there

**Review by C-Flatla — APPROVED** [review]
Code changes make sense and behave correctly. Not too picky about the indentation so feel free to leave as is or make the change 😄 

LGTM

**Comment by cwtedwards** [comment]
@C-Flatla I'm thinking I will fix the indentation with the PR I am working on for the ellipses, so we can get this merged if you're cool with that

**Comment by C-Flatla** [comment]
@cwtedwards sounds good to me 🚀

**Pull request merged**